src/prodam/secoes/profiles/default/types/service.xml

```diff
@@ -2,7 +2,7 @@
 <object name="service" meta_type="Dexterity FTI" i18n:domain="plone"
    xmlns:i18n="http://xml.zope.org/namespaces/i18n">
  <property name="title" i18n:translate="">Service</property>
- <property name="description" i18n:translate="">None</property>
+ <property name="description" i18n:translate="">Add a a service</property>
```

Comment:
O que vc quis dizer aqui? (e tem dois a)
Se for: Adiciona uma notícia = Adds a service
Mas a descrição é do tipo de item ou do ato de adicionar? Se for do tipo de item pode ser uma descrição genérica, não? A service that will be filtered by/in the search